Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -62,19 +59,40 @@ public function getConfig()
                 $configs['card_brands'] = $this->capabilities->getCardBrands();
             }
 
-            // filter only active backends
-            if (array_key_exists($code, $backends)) {
-                if ($code === 'omise_offsite_shopeepay') {
-                    $configs['omise_payment_list'][$code] = $this->getShopeeBackendByType($backends[$code]);
-                } else {
-                    $configs['omise_payment_list'][$code] = $backends[$code];
-                }
-            }
+            $this->filterActiveBackends($code, $configs['omise_payment_list']);
         }
 
         return $configs;
     }
 
+    /**
+     * filter only active backends
+     * @param $code         Payment method code
+     * @param $paymentList  Reference of the payment list
+     */
+    private function filterActiveBackends($code, &$paymentList)
+    {
+        // Retrieve available backends & methods from capabilities api
+        $backends = $this->capabilities->getBackendsWithOmiseCode();
+        $tokenization_methods = $this->capabilities->getTokenizationMethodsWithOmiseCode();
+        $backends = array_merge($backends, $tokenization_methods);
```

Comment:
@aashishgurung why shadow the `$backends` variable?

Reply:
Here, we are just merging  two arrays, `$backends` and `$tokenization_methods`, to `$backends`. Our capability API returns payment methods in two different object. One under payment methods and another under tokenization methods. For clarity, I will rename the variable names.